src/mlpack/tests/CMakeLists.txt

```diff
@@ -3,31 +3,53 @@ include(CTest)
 # mlpack test executable.
 add_executable(mlpack_test
   EXCLUDE_FROM_ALL
-  activation_functions_test.cpp
+  # Tests for neural network code (and reinforcement learning code).
+  ann/activation_functions_test.cpp
+  ann/ann_dist_test.cpp
+  # This includes all files in ann/layer/.
+  ann/layer_test.cpp
+  ann/ann_regularizer_test.cpp
+  ann/ann_test_tools.hpp
+  ann/convolutional_network_test.cpp
+  ann/convolution_test.cpp
+  ann/custom_layer.hpp
+  ann/feedforward_network_test.cpp
+  ann/init_rules_test.cpp
+  ann/ksinit_test.cpp
+  ann/loss_functions_test.cpp
+  ann/recurrent_network_test.cpp
+
+  ann/async_learning_test.cpp
+  ann/augmented_rnns_tasks_test.cpp
+  ann/reward_clipping_test.cpp
+  ann/rl_components_test.cpp
+
+#  ann/not_adapted/activation_functions_test.cpp
```

Comment:
Perhaps a good idea to comment on this? The name gives it somewhat away, so not sure it's necessary.

Reply:
Yeah, agreed, done in e5e8829ea.